srcpkgs/pulseeffects-legacy/files/README.voidlinux

```diff
@@ -0,0 +1,3 @@
+Optional dependencies:
+- lsp-plugins (compressor, equalizer, delay, loudness compensator)
+- zam-plugins (maximizer)
```

Comment:
I am not sure if this is necessary, all of the information here is mentioned in the program

Reply:
This isn't necessary and zam-plugins doesn't even exist.